examples/lock-app/silabs/include/LockManager.h

```diff
@@ -199,10 +199,12 @@ class LockManager
 private:
     struct UnlatchContext
     {
+        static constexpr uint8_t kMaxPinLenght = UINT8_MAX;
```

Comment:
should this be a size_t like the mPinLength member?

Reply:
Three reasons:

1.  size_t is 32 bits, so it takes more space.
2. I don't really see a need for more than 255 characters to a doorlock pin for our sample app.
3. Since it's in our app code, a client that has a setup with a longer pin can change that part of the code somewhat easily.

Reply:
in that case, should  mPinLength be of the same type of this? The max is a uint8 but the variable that stores the information is a uint32?

Reply:
indeed

Reply:
changed to uint8_t